readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -142,6 +143,19 @@ def get_config_params(self):
                     self.project.slug, self.version.slug,
                 )
 
+        build_id = self.build_env.build.get('id')
+        build_url = None
+        if build_id:
+            build_url = reverse(
+                'builds_detail',
+                kwargs={
+                    'project_slug': self.project.slug,
+                    'build_pk': build_id,
+                },
+            )
+            protocol = 'http' if settings.DEBUG else 'https'
+            build_url = f'{protocol}://{settings.PRODUCTION_DOMAIN}{build_url}'
```

Comment:
Is this not the same as `build.get_full_url`?

Reply:
We don't have the build object here (no db access), we have a dictionary from the api response.